addons/hr_timesheet/views/hr_timesheet_views.xml

```diff
@@ -63,6 +63,30 @@
             </field>
         </record>
 
+        <record id="view_hr_timesheet_line_pivot_groupby_project" model="ir.ui.view">
```

Comment:
Shouldn't we move the records you are adding here and that only used in `addons/hr_timesheet/report/hr_timesheet_report_view.xml` into the same file?

Reply:
@IT-Ideas , Agree, done it. Thanks